keyboards/ymdk/sp64/keymaps/via/keymap.c

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2021 Your Name (@yourgithub)
```

Comment:
Needs to be filled out.

Reply:
Ups, my bad, I copy-pasted without reading. Adressed in https://github.com/qmk/qmk_firmware/pull/16152/commits/5360cbb2348d097cbaefb984d288ed576d9ff390